README.md

```diff
@@ -1,13 +1,11 @@
 # Rusqlite
 
-[![Travis Build Status](https://api.travis-ci.org/rusqlite/rusqlite.svg?branch=master)](https://travis-ci.org/rusqlite/rusqlite)
-[![AppVeyor Build Status](https://ci.appveyor.com/api/projects/status/github/rusqlite/rusqlite?branch=master&svg=true)](https://ci.appveyor.com/project/rusqlite/rusqlite)
-[![Build Status](https://github.com/rusqlite/rusqlite/workflows/CI/badge.svg)](https://github.com/rusqlite/rusqlite/actions)
-[![dependency status](https://deps.rs/repo/github/rusqlite/rusqlite/status.svg)](https://deps.rs/repo/github/rusqlite/rusqlite)
```

Comment:
Could you please keep the deps status ?

Reply:
Sure. It's inaccurate (we aren't insecure — even if you turn on the `chrono` dep, we disable the feature that is vulnerable...), and I'd prefer that we use something like a GHA check for `cargo audit` or something anyway, but I'll try and make that case another time.